ai-platform-unified/notebooks/official/pipelines/pipelines_intro_kfp.ipynb

```diff
@@ -0,0 +1,1109 @@
+{
```

Comment:
<p>"with three steps"</p><p>Was not clear to me on first reading if pipeline consists of 3 steps, or takes 3 steps to make the pipeline.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/335/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>